packages/client/tests/e2e/_utils/run.ts

```diff
@@ -135,12 +109,8 @@ async function main() {
   }
 
   const dockerVolumes = [
-    `/tmp/prisma-0.0.0.tgz:/tmp/prisma-0.0.0.tgz`,
-    `/tmp/prisma-debug-0.0.0.tgz:/tmp/prisma-debug-0.0.0.tgz`,
-    `/tmp/prisma-client-0.0.0.tgz:/tmp/prisma-client-0.0.0.tgz`,
-    `/tmp/prisma-engines-0.0.0.tgz:/tmp/prisma-engines-0.0.0.tgz`,
-    `/tmp/prisma-generator-helper-0.0.0.tgz:/tmp/prisma-generator-helper-0.0.0.tgz`,
-    `/tmp/prisma-nextjs-monorepo-workaround-plugin-0.0.0.tgz:/tmp/prisma-nextjs-monorepo-workaround-plugin-0.0.0.tgz`,
```

Comment:
That's a really nice cleanup 💚 

It would help for https://github.com/prisma/team-orm/issues/595